src/main/java/com/googlecode/protobuf/format/JsonJacksonFormat.java

```diff
@@ -144,9 +145,12 @@ public void merge(JsonParser parser,
     }
 
 
-
     protected JsonGenerator createGenerator(OutputStream output) throws IOException {
-    	JsonGenerator generator = jsonFactory.createJsonGenerator(output, JsonEncoding.UTF8);
+        return createGenerator(output, Charset.forName(JsonEncoding.UTF8.getJavaName()));
```

Comment:
Hmm… If you know you want UTF8, can you just use `StandardCharsets.html.UTF_8`?
https://docs.oracle.com/javase/8/docs/api/java/nio/charset/StandardCharsets.html#UTF_8